pom.xml

```diff
@@ -630,6 +630,37 @@
         </configuration>
       </plugin>
 
+      <!--
+        ~ Memento rules to remove overrides from child modules, whenever the changes are moved to `logging-parent`.
+        ~
+        ~ When empty, please comment this execution out, but do not delete it!
+        -->
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-enforcer-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>remove-overrides</id>
+            <goals>
+              <goal>enforce</goal>
+            </goals>
+            <!-- Wait until we retrieve `parent.version` property in `build-helper-maven-plugin` below -->
+            <phase>initialize</phase>
+            <inherited>false</inherited>
+            <configuration>
+              <rules>
+                <requireProperty>
+                  <property>parent.version</property>
+                  <regex>12\.1\.1</regex>
+                  <regexMessage>Remember to remove overrides from child modules, whenever the changes are moved to `logging-parent`:
+                    * Remove the override `flatten-maven-plugin` configuration from `log4j-parent/pom.xml`.</regexMessage>
```

Comment:
Why do we do this? Are we intending to move this _"non-BOM flattening"_ logic to `logging-parent`?

Reply:
You're right. Could you then instead move this PR to `logging-parent` instead, please? We have full control over both projects and their releases. Maybe I'm missing out on certain details, but I am not able to see why we shall temporarily fix this only for Log4j, whereas fixing it in `logging-parent` will benefit all its children.